specutils/manipulation/utils.py

```diff
@@ -0,0 +1,154 @@
+from __future__ import division
+
+import numpy as np
+
+from ..spectra import Spectrum1D, SpectralRegion
+
+__all__ = ['excise_regions']
```

Comment:
probably `linear_exciser` needs to be here since it's part of the public API by way of `excise_regions` accepting it as an argument

Reply:
added...